uniuri.go

```diff
@@ -112,3 +112,14 @@ func NewLenChars(length int, chars []byte) string {
 		}
 	}
 }
+
+// NewLenChars returns a new random string of the provided length, consisting
+// of the provided byte slice of allowed characters (maximum 256).
+func NewLenChars(length int, chars []byte) string {
+	b := NewLenCharsBytes(length, chars)
```

Comment:
I think this can be just `return string(NewLenCharsBytes(length, chars))`, since nil []byte is converted to empty string.

Reply:
Totally agree, sorry I forgot about that :smiley: .